test/documentSymbols.test.ts

```diff
@@ -298,6 +298,28 @@ suite('Document Symbols Tests', () => {
             );
         });
 
+        it('Document Symbols with complex mapping and aliases', () => {
+            const content = `version: 0.0.1
+            structure:
+              ? &root root
+              :
+                element: div
+            conditions:
+              ? *root
+              :
+                style:
+                  height: 41
+            `;
+
+            try {
+                const symbols = parseHierarchicalSetup(content);
+                assert.equal(symbols.length, 2);
```

Comment:
Can you check the actual symbols themselves so that we know we are getting the symbols back with the correct start and end etc, similiar to the other tests